adafruit_as726x.py

```diff
@@ -258,12 +267,12 @@ def conversion_mode(self, val):
         self._virtual_write(_AS726X_CONTROL_SETUP, state | (val << 2))
 
     @property
-    def gain(self):
+    def gain(self) -> int:
         """The gain for the sensor"""
         return self._gain
 
     @gain.setter
-    def gain(self, val):
+    def gain(self, val: int) -> None:
```

Comment:
It looks like this could potentially be `3.7` so it seems like it should be `float`

Reply:
I think you're right. Adjusted.